src/index.ts

```diff
@@ -21,7 +21,8 @@ export * from './shared/';
 export * from './security/';
 export * from './subscriptions/';
 export * from './supportCenter/';
-export * from './securityScore/';
+// export * from './securityScore/';
```

Comment:
probably break the compatibility with old other consumers of the project.
TO REVERT